### PR TITLE
Fetch stderr and stdout for action result

### PR DESCRIPTION
**Pull request by luluz66**
Screenshot for an failed action:
Screen Shot 2022-01-03 at 1 06 59 PM

### CODE REVIEW

File: app/invocation/invocation_action_card.tsx [hunk 1]
```diff
@@ -86,10 +91,47 @@ export default class InvocationActionCardComponent extends React.Component<Props
         this.setState({
           actionResult: build.bazel.remote.execution.v2.ActionResult.decode(new Uint8Array(buffer)),
         });
+        this.fetchStdoutAndStderr(this.state.actionResult);
       })
       .catch((e) => console.error("Failed to fetch action result:", e));
   }
 
+  fetchStdoutAndStderr(actionResult: build.bazel.remote.execution.v2.ActionResult) {
+    console.log("fetch stdout and stderr ");
```
Comment (siggisim) [inline app/invocation/invocation_action_card.tsx:100]: rm

**Review by siggisim — APPROVED** [review]
This is awesome!

File: app/invocation/invocation_action_card.tsx (hunk 1 above)
Comment (luluz66) [reply app/invocation/invocation_action_card.tsx]: Done.

**Review by bduffany — APPROVED** [review]
Nice :)

**Pull request merged**